proto/astarteplatform/msghub/property.proto

```diff
@@ -0,0 +1,62 @@
+/*
+ * This file is part of Astarte.
+ *
+ * Copyright 2024 SECO Mind Srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+syntax = "proto3";
+
+package astarteplatform.msghub;
+
+import "astarteplatform/msghub/astarte_type.proto";
+
+/*  */
+enum Ownership {
+  DEVICE = 0;
+  SERVER = 1;
+}
+
+/*  */
+message Property {
```

Comment:
Have you considered to reuse ` AstarteMessage` from `astarte_message.proto`?

Reply:
The problem I see is that the message you mention contains other fields that I don't think we need:
- mainly the interface_name which is known from the containing object or the request made.
- timestamp which is not specified for properties.
It also allows the use of objects, which could be a good thing for future proofing the APIs (maybe). I'm not sure but maybe @rgallor and @joshuachp have other clues.

Reply:
Yes, I agree with you. It doesn't seem all the fields are necessary in our scenarios